witness/README.md

```diff
@@ -1,15 +1,45 @@
-Witnessing
-============
+# Log Witnessing
 
-Witnessing is an important part of any transparent ecosystem, as it ensures that
-misbehavior on the part of logs can be prevented or detected. This is done by
-ensuring that the _checkpoints_ published by logs are _consistent_, which means 
-the log is presenting the same version of its contents to everyone and
-thus isn't carrying out a _split-view attack_.
+## Overview
 
-In terms of checkpoints, we consider witness implementations that use the
+Witnessing is an important part of log ecosystems that prevents logs from
+successfully accomplishing a _split-view attack_. A split-view attack is
+one in which the log prevents different populations with different views.
+Each of these views is append-only, and thus all clients checking consistency
```

Comment:
I would say that "Each of these views *may be* append-only" and then "all clients [...] *would be* satisfied", since a (somewhat dumb) log could even provide inconsistent checkpoints to the same population (like for example one that it knows doesn't check consistency proofs)?